projects/fsastorefrontlib/src/cms-components/checkout/components/quote-review/quote-review.component.html

```diff
@@ -37,6 +37,13 @@ <h2 class="row no-gutters section-header-heading">
                 [opened]="true"
                 [title]="'fscommon.oboCustomers' | cxTranslate"
               >
+                <div class="position-absolute w-100 message-position">
+                  <cx-fs-message
+                    [messageText$]="oboCustomerMessage$"
```

Comment:
Can we just pass in `quoteReview.message.selectedOboCustomer | cxTranslate` here, thus avoiding using `translationService` in model?

Reply:
I could, but then I will have difficulties setting debounceTime and react every time input changes. Maybe I need then to use ChangeDetectionRef. But why is it a problem to use already existing `translationService ` in the model, don't understand?